hi-ml-multimodal/src/health_multimodal/__init__.py

```diff
@@ -2,3 +2,21 @@
 #  Copyright (c) Microsoft Corporation. All rights reserved.
 #  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 #  -------------------------------------------------------------------------------------------
+
+BIOMED_VLP_CXR_BERT_SPECIALIZED = "microsoft/BiomedVLP-CXR-BERT-specialized"
+REPO_URL = f"https://huggingface.co/{BIOMED_VLP_CXR_BERT_SPECIALIZED}"
+CXR_BERT_COMMIT_TAG = "v1.0"
```

Comment:
We should probably pull the same version of the text and image models - maybe we can create a new tag for the image model release?

Reply:
@Shruthi42 : Thanks for raising this. We created a common tag for both models.

Reply:
@ozan-oktay has created `v1.1`. I'll update in a minute.